### DISCUSSION TITLE
Email report for all my sites in one email

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#617] [Anything Else]
Similar to the /sites screen that just shows a list of all sites I have in Plausible, number of unique visitors, number of pageviews and comparison to the previous period. One line for each site. Sent via email like the regular weekly/monthly reports.

(feature request via email)

**Comment by BrunoBernardino** [comment]
I'm happy to help build this if I can get some pointers on where to look at!

Once you have more than a handful of websites, it becomes quite noisy to receive the reports for all of them.
